lib/autoload.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+class ClassLoader {
+    private $searchPath = [];
```

Comment:
@Frenzie Do you know why Travis did not catch the space indenting here?
@aledeg Could you change to tabs like the other files?

Reply:
Isn't lib/* excluded or something? Please ping me in a couple of days if I forget to check.

Reply:
Not /*. Strange.
https://github.com/FreshRSS/FreshRSS/blob/0a2d9b3b54ee51a3965d79c68409ef045ad5834b/phpcs.xml#L6-L9

Reply:
I've changed to tabs.